selfdrive/test/test_onroad.py

```diff
@@ -98,6 +100,13 @@
   "wideRoadCameraState": [1.5, 0.35],
 }
 
+LOGS_SIZE_RATE = {
+  "qlog": 0.0083,
+  "rlog": 0.1528,
+  "qcamera.ts": 0.03828,
```

Comment:
I assume these are per second? these should still be on the same level to keep them readable

Reply:
Yes per second. 

Do you mean the hevc ones?

Reply:
I mean all of them. I know a qlog is around 500K, but now there's some new unit I need to convert to